### PR TITLE
Migrate charts job to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @foxish

### CODE REVIEW

**Comment by foxish** [comment]
/lgtm

Thank you!

**Comment by krzyzacy** [comment]
take another look - moved to prow, and fixed up some path. Using :latest kubetest image for now so that I don't have to push a new image for prod for the kubetest changes, will use a stable tag once the job becomes green.

File: kubetest/e2e.go [hunk 1]
```diff
@@ -495,7 +495,7 @@ func perfTest() error {
 
 func chartsTest() error {
 	// Run helm tests.
-	if err := finishRunning(exec.Command("/src/k8s.io/charts/test/helm-test-e2e.sh")); err != nil {
+	if err := finishRunning(exec.Command("%s/src/k8s.io/charts/test/helm-test-e2e.sh", os.Getenv("GOPATH"))); err != nil {
```
Comment (foxish) [inline kubetest/e2e.go]: Is $GOPATH set automatically to `/go/`?

Comment (krzyzacy) [reply kubetest/e2e.go]: yeah, from base image that's `/go`, you can also add a customized $GOPATH from prow config

**Comment by foxish** [comment]
/lgtm

Thanks @krzyzacy

**Comment by krzyzacy** [comment]
(building a new :latest image, will merge when that finishes)

**Pull request merged**